apps/client/src/app/pages/accounts/transfer-cash-balance/transfer-cash-balance.module-dialog.ts

```diff
@@ -0,0 +1,27 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { MatButtonModule } from '@angular/material/button';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatDialogModule } from '@angular/material/dialog';
+import { MatFormFieldModule } from '@angular/material/form-field';
+import { MatInputModule } from '@angular/material/input';
+import { MatSelectModule } from '@angular/material/select';
+
+import { TransferCashBalanceDialog } from './transfer-cash-balance-dialog.component';
+
+@NgModule({
+    declarations: [TransferCashBalanceDialog],
+    imports: [
+        CommonModule,
+        FormsModule,
+        MatButtonModule,
+        MatCheckboxModule,
```

Comment:
Not used, please remove

Reply:
Hi @dtslvr the MatButtonModule and CommonModule are used in the html part.